BUILD.md

```diff
@@ -111,6 +111,25 @@ If you want to build with an earlier version, you must temporarily remove the 'C
 ### MKL-DNN
 To build ONNX Runtime with MKL-DNN support, build it with `./build.sh --use_mkldnn --use_mklml`
 
+### Tensor RT
+ONNX Runtime supports the Tensort RT execution provider (released as preview). You will need to download and install [CUDA](https://developer.nvidia.com/cuda-toolkit), [CUDNN](https://developer.nvidia.com/cudnn) and [TensorRT](https://developer.nvidia.com/nvidia-tensorrt-download).
+
+The TensorRT execution provider for ONNX Runtime is built and tested with CUDA 10.01, CUDNN 7.1 and TensorRT 5.0.2.6. 
+
+ - The path to the CUDA installation must be provided via the CUDA_PATH environment variable, or the `--cuda_home parameter`.
+ - The path to the CUDNN installation (include the `cuda` folder in the path) must be provided via the CUDNN_PATH environment variable, or `--cudnn_home parameter`. The CUDNN path should contain `bin`, `include` and `lib` directories.
+ - The path to the CUDNN bin directory must be added to the PATH environment variable so that cudnn64_7.dll is found.
+
+- The path to TensorRT installation must be provided via the TRT_PATH environment variable or the `--tensorrt_home parameter`.
+
+You can build from source on Linux by using the following cmd from the onnxruntime directory:
+
+```
+./build.sh --cudnn_home /usr --cuda_home /usr/local/cuda --usr_tensorrt --tensorrt_home <path to TensorRT home> (Linux)
```

Comment:
for cudnn_home , can you use the default install location for cudnn?

Reply:
updated to /usr/lib/x86_64-linux-gnu/